src/simulation.hpp

```diff
@@ -576,6 +578,33 @@ template <typename problem_t> void AMRSimulation<problem_t>::setInitialCondition
 		ReadCheckpointFile();
 	}
 
+#if AMREX_SPACEDIM == 3
```

Comment:
You need to add a conditional here so that it only does this if gravity is enabled. Otherwise, it doesn't make sense. I think that's what is causing the test failures.